pyproject.toml

```diff
@@ -1,17 +1,18 @@
 [project]
 name = "owasp-depscan"
-version = "4.2.9"
+version = "5.0.0"
```

Comment:
Shall we call this 4.5.0 so that we try to include reachability analysis to v5?

Reply:
@prabhu Sure.